pandas/tseries/offsets.py

```diff
@@ -721,6 +724,7 @@ def apply(self, other):
 
             return result
         else:
+            # TODO: Figure out the end of this sente
```

Comment:
I presume you're going to figure this out beforehand?

Reply:
You mean what the end of the error message should be?  That's orthogonal to this PR, but merits a reminder.

Reply:
yeah not sure, @sinhrks wrote this originally.